### PR TITLE
s is the second commit

### PR DESCRIPTION
**Pull request by wazedrahib90**

### CODE REVIEW

**Comment by wazedrahib90** [comment]
good

File: src/test/java/com/paxovision/steps/Mysteps.java [hunk 1]
```diff
@@ -46,8 +46,7 @@ public void browse_to_the_application() throws Throwable {
 
 		@Then("^Broadleaf home page should show$")
 		public void broadleaf_home_page_should_show() throws Throwable {
-		 
-			heatclinic.homePage().showloginPage();
```
Comment (wazedrahib90) [inline src/test/java/com/paxovision/steps/Mysteps.java]: cool

Comment (wazedrahib90) [inline src/test/java/com/paxovision/steps/Mysteps.java]: good

**Comment by malikirum365** [comment]
what is this bhai?

Thanks & Regards

Irum Malik

Cell    : (347) 486-4005
Email : malikirum365@gmail.com

**Pull request closed without merging**